templates/zerver/help/configure-desktop-notifications.md

```diff
@@ -0,0 +1,21 @@
+# Configure desktop notifications
+To change your desktop notification settings, click on the cog
+(![cog](/static/images/help/cog.png)) in the top right corner.
+Click on **Settings** in the menu that pops up.
+
+![Settings](/static/images/help/cog_dropdown_settings.png)
+
+You will first see the **Your Account** section.
+Scroll down to the **Notifications** section.
+
+![Notification Settings](/static/images/help/notification-settings.png)
+
+If you want notifications for each new message from a stream, check the
+desktop notifications box under **Stream messages**.
+
+![Stream notifications](/static/images/help/stream-notification-settings.png)
+
+If you want notifications for private messages and @-mentions, check the
+desktop notifications box under **Private messages and @-mentions**.
+
```

Comment:
Hey, I just might be OCDing a bit but I think you should add a line about the Save button at the end.